tesseract/tesseract_scene_graph/examples/parse_srdf_example.cpp

```diff
@@ -59,4 +59,5 @@ int main(int /*argc*/, char** /*argv*/)
 
   AllowedCollisionMatrix::ConstPtr acm = g->getAllowedCollisionMatrix();
   const AllowedCollisionMatrix::AllowedCollisionEntries& acm_entries = acm->getAllAllowedCollisions();
+  CONSOLE_BRIDGE_logInform("ACM Number of entries: %d", acm_entries.size());
```

Comment:
Just making sure this was intended for this PR. It doesn't look like a Clang fix.

Reply:
It was giving an unused variable warning for acm_entries.

Reply:
Yea it is due to the ifdef.